{"name": "test_Once_logged_in", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../config.json'", "trace": "self = <tests.main_test_home.TestHome testMethod=test_Once_logged_in>\n\n    def setUp(self):\n>       f = open('../config.json', 'r')\nE       FileNotFoundError: [Errno 2] No such file or directory: '../config.json'\n\ntests/main_test_home.py:11: FileNotFoundError"}, "start": 1681672202930, "stop": 1681672202930, "uuid": "3c332ea9-60e2-459e-ac01-c4e7eb2d2624", "historyId": "f652da2ff8fd24b52e6a085e67e6276f", "testCaseId": "16a8fd6c06b249642ccc8cdb1d99606f", "fullName": "main.TestHome#test_Once_logged_in", "labels": [{"name": "suite", "value": "main"}, {"name": "subSuite", "value": "TestHome"}, {"name": "host", "value": "Siwars-MacBook-Pro.local"}, {"name": "thread", "value": "57983-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "main"}]}